Sources/SWBCore/SpecImplementations/LinkerSpec.swift

```diff
@@ -101,6 +103,7 @@ open class LinkerSpec : CommandLineToolSpec, @unchecked Sendable {
             self.dsymPath = dsymPath
             self.xcframeworkSourcePath = xcframeworkSourcePath
             self.privacyFile = privacyFile
+            self.libPrefix = prefixes.first
```

Comment:
Why do we only consider the first element here?

Reply:
Only handling one sounds ok to me, but I think we should then either only allow one in the spec or error out if we parse more than one